lms/djangoapps/student_account/test/test_views.py

```diff
@@ -1,8 +1,8 @@
 # -*- coding: utf-8 -*-
 """ Tests for student account views. """
 
-import re
 from nose.plugins.attrib import attr
```

Comment:
Nit: Import order: nose is not in the standard library, so it should go in the second section (in alphabetical order) with all the django imports.  Also, `from copy import copy` should be up here with the standard library imports.